app/parse.py

```diff
@@ -15,9 +31,102 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+options = webdriver.ChromeOptions()
+options.add_argument("--headless")
+driver = webdriver.Chrome(options=options)
```

Comment:
The `driver` instance is created globally, which can lead to issues if the script is run multiple times or in parallel. Consider encapsulating the driver initialization and termination within a function or context manager to ensure proper resource management.